content/docs/reference/Releases/regolith-3.3-release-notes.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "3.3 Release Notes"
+linkTitle: "3.3 Release Notes"
+weight: 84
+description: >
+  Release notes for Regolith 3.3
+prev: /docs/reference/releases
+---
+
+## Regolith 3.3 Release Notes
+
+Regolith 3.3 is a minor release focusing on Ubuntu 25.04 support, numerous build system improvements, and sway 1.10.
+
+## Features
+
+* Ubuntu 25.04 (Plucky) support
```

Comment:
Due to delay, adding the support for Ubuntu 24.10 is also being shipped in this release.